app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -6,6 +6,10 @@ def call(toolbar_name)
     build_toolbar(toolbar_name)
   end
 
+  def call_by_class(toolbar_class)
+    build(toolbar_class)
+  end
```

Comment:
Well "randomly". It's not randomly once you need it.

And I'd rather have it randomly public that have 2 methods that do the same one public and the other private. That looks wrong.